### PR TITLE
feat(i18n): Russial localization improvements

### PR DESCRIPTION
**Pull request by birksovskiy**
I'm really lazy to write something here.

### CODE REVIEW

File: locales/ru-RU.json [hunk 1]
```diff
@@ -34,10 +34,14 @@
...
     "posts_count": "{0} Пост|{0} Поста|{0} Постов|{0} Постов",
     "profile_description": "Заголовок профиля {0}",
+    "profile_personal_note": "Заметка к профилю",
     "profile_unavailable": "Профиль недоступен",
+    "request_follow": "Запрос на подписку",
     "unblock": "Перестать блокировать",
     "unfollow": "Перестать читать",
-    "unmute": "Перестать игнорировать"
+    "unmute": "Перестать игнорировать",
+    "view_other_followers": "Подписчики с других экземпляров не могут быть показаны",
```
Comment (sheremet-va) [inline locales/ru-RU.json:43]: There is already a translation for "instances" in `timeline.view_older_posts`: "инстансы" - I don't actually know a good translation for this either way, but "экземпляры" doesn't sound right

File: locales/ru-RU.json [hunk 2]
```diff
@@ -159,7 +164,24 @@
...
+    "clear_error": "Ошибка очистки",
+    "create": "Создать",
+    "delete": "Удалить список",
+    "delete_error": "Произошла ошибка при удалении списка",
+    "edit": "Редактировать список",
+    "edit_error": "Произошла ошибка при изменении списка",
+    "error": "Произошла ошибка при создании списка",
+    "error_prefix": "Ошибка: ",
+    "list_title_placeholder": "Название списка",
+    "modify_account": "Изменить список с помощью аккаунта",
```
Comment (sheremet-va) [inline locales/ru-RU.json:179]: Where is this actually used? The translation seems suspicious 🤔

**Review by sheremet-va — CHANGES_REQUESTED** [review]

File: locales/ru-RU.json (hunk 2 above)
Comment (birksovskiy) [reply locales/ru-RU.json:179]: I don't understand "modify_account"'s purpose. I haven't found where is it.

**Comment by birksovskiy** [comment]
@sheremet-va I've resolved your suggestions

**Review by sheremet-va — APPROVED** [review]

**Comment by birksovskiy** [comment]
It isn't a link to pull request #2, it is my mistake

**Comment by patak-dev** [comment]
Thanks @birksovskiy!

**Pull request merged**